vlm/src/main/scala/geotrellis/contrib/vlm/GeoTiffReprojectRasterSource.scala

```diff
@@ -94,6 +94,9 @@ class GeoTiffReprojectRasterSource(
     new GeoTiffReprojectRasterSource(uri, targetCRS, options)
 
   def resample(resampleGrid: ResampleGrid, method: ResampleMethod): RasterSource =
-    new GeoTiffResampleRasterSource(uri, resampleGrid, method)
+    new GeoTiffResampleRasterSource(uri, resampleGrid, method) {
```

Comment:
Thats why i had metadata mismatch. In fact chained operations didn't work. Chained means `RasterSource.reproject.resample... etc`

Reply:
`GeoTiffReprojectRasterSource!.resample` should produce `GeoTiffReprojectRasterSource` with modified rasterExtent, not `GeoTiffResampleRasterSource`